### PR TITLE
MCPE V1.0.4 with New protocol + packet ids

### PR DESCRIPTION
**Pull request by AppleDevelops**
## PR Description
1.0.4 beta is out already, and here I am ready to update us. Sadly it's really late right now, and I'm heading off to bed, I'll finish this stuff tomorrow. 

## Have you tested it?
- [ ] I have tested it.
- [x] I have not tested it.
...

### CODE REVIEW

**Comment by remotevase** [comment]
Really they forgot protocol bump? Wow thats usually what they do **first!** XD

**Comment by daniktheboss** [comment]
Lmao.... nice pie

**Comment by remotevase** [comment]
Tested @AppleDevelops

**Comment by remotevase** [comment]
Also, [this](https://twitter.com/MCPEAppStore) will notify you when 1.0.4 is released on app store. On play store already technically released as beta, but app store is when we add to master

File: src/pocketmine/inventory/PlayerInventory.php [hunk 1]
```diff
@@ -225,7 +225,7 @@ public function sendHeldItem($target){
 		$item = $this->getItemInHand();
 
 		$pk = new MobEquipmentPacket();
-		$pk->eid = ($target === $this->getHolder() ? 0 : $this->getHolder()->getId());
+		$pk->eid = $this->getHolder()->getId();;
```
Comment (Sandertv) [inline src/pocketmine/inventory/PlayerInventory.php]: *facepalm*

Comment (AppleDevelops) [inline src/pocketmine/inventory/PlayerInventory.php]: Derp lol

**Pull request merged**